db/migrate/20200127093938_create_roles.rb

```diff
@@ -0,0 +1,8 @@
+class CreateRoles < ActiveRecord::Migration[5.2]
+  def change
+    create_table :roles do |t|
+      t.string :name
+      t.timestamps
+    end
+  end
+end
```

Comment:
@sayalisheth Check how to add the default roles in migration. SuperAdmin, Admin, Passenger

Reply:
Done